tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
> but if I prefix it with a backslash it defaults back to the global namespace as intended. If that works

I would argue that this can be consider as a bug (but that just me). Everything under a namespaced `Route::group` should append the suggested namespace.

And even if you want the framework to support such option, deleting a testcase that test something else instead of creating new testcase to support your suggestion and the same time doesn't break existing code is the way to go.